refactor(curriculum): improve binary search tree (#53622)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/07-scientific-computing-with-python/learn-tree-traversal-by-building-a-binary-search-tree/65c6409418806da8d0636ffc.md

---
id: 65c6409418806da8d0636ffc
title: Step 26
challengeType: 20
dashedName: step-26
---

# --description--

The `insert` and `search` functionalities are complete, it's time to test them.

> Note that, at this point, the nodes are not sorted and just inserted. We will work on sorting using inorder traversal later on.

Reviewer: ```suggestion
Note that, at this point, the nodes are not sorted and just inserted. You'll work on sorting using inorder traversal later on.
```
